### PR TITLE
ACF Template Targeting 

### PR DESCRIPTION
**Pull request by kylehotchkiss**
re #91

### CODE REVIEW

File: wordpress/composer.json [hunk 1]
```diff
@@ -63,7 +63,7 @@
     "wpackagist-plugin/term-management-tools": "2.0.1",
     "wpackagist-plugin/wordpress-seo": "16.3.0",
     "wpackagist-plugin/wp-graphql": "1.3.8",
-    "wp-graphql/wp-graphql-acf": "0.4.0",
+    "wp-graphql/wp-graphql-acf": "0.5.0",
```
Comment (ccorda) [inline wordpress/composer.json:66]: 0.5.3 is the latest version

**Review by ccorda — APPROVED** [review]
One thing that I think we can remove as part of this is:
`function expose_acf_to_graphql_only()` in `wordpress/wp-content/themes/headless/setup/helpers/wpgraphql.php`. That was an old workaround I don't believe we need any more.

File: wordpress/wp-content/themes/headless/acf-json/group_5f4f3a7bdc981.json [hunk 2]
```diff
@@ -58,5 +58,9 @@
     "hide_on_screen": "",
     "active": true,
     "description": "",
-    "modified": 1599028053
+    "show_in_graphql": 0,
+    "graphql_field_name": "flexBlockquote",
```
Comment (ccorda) [inline wordpress/wp-content/themes/headless/acf-json/group_5f4f3a7bdc981.json:62]: small thing, but this one doesn't need a graphql field name (the others are empty strings)

**Comment by kylehotchkiss** [comment]
Removing `expose_acf_to_graphql_only()` locally worked for me. Verified by running `yarn build`. Vercel builds showing failure because the graphql-acf plugin needs to be deployed to Wordpress still. Since this is approved, i'm going to merge it and deploy the updates to wordpress now.

**Pull request merged**